docs/README.md

```diff
@@ -7,7 +7,7 @@ Installing OpenFaaS Cloud requires some initial setup with GitHub or GitLab and
 * Kubernetes or Docker Swarm (Docker cannot be running with XFS as a backing file-system due to buildkit restrictions)
 * Registry account - Docker Hub account or private registry with TLS
 * OpenFaaS deployed with authentication enabled
-* Extended timeouts for the queue-worker, gateway and the backend provider
```

Comment:
What are the recommended timeouts? Can we give some pointers here? example config etc... I only see something for the queue-worker 👇